src/core/getFrameAccountAddress.ts

```diff
@@ -26,16 +27,10 @@ async function getFrameAccountAddress(
   // Get the Farcaster ID from the message
   const farcasterID = validatedMessage?.data?.fid ?? 0;
   // Get the user verifications from the Farcaster Indexer
-  const options = {
-    method: 'GET',
-    url: `https://api.neynar.com/v2/farcaster/user/bulk?fids=${farcasterID}`,
-    headers: { accept: 'application/json', api_key: NEYNAR_API_KEY },
-  };
-  const resp = await fetch(options.url, { headers: options.headers });
-  const responseBody = await resp.json();
-  // Get the user verifications from the response
-  if (responseBody.users) {
-    const userVerifications = responseBody.users[0] as FidResponse;
+  const neynarClient = new NeynarClient();
```

Comment:
I wonder if we should create a new instance of `NeynarClient()` everytime `getFrameAccountAddress` is called.

So far it seems we are doing:
- someone might use `getFrameAccountAddress` multiple times
- each time a new `neynarClient` instance is created from `NeynarClient`
- each time a new `this.user` instance is created from `NeynarUserFunction(apiKey);`

So my personal feeling after reading this three times, I think we should avoid using Classes just because:
1. anytime `new` is run in JS it has some hidden memory and cpu cost that I would like to avoid from our side
2. all the data generated it feels gated inside an instance
3. the user will abuse our funtion, so every small code weakness will become extra big publically, so beacuse 1 and 2, this feels a bit worry to me.

@robpolak would you be ok to rewrite this as Functional-oriented instead of Class-oriented?
Also as mention over our sync, let's use `utils` as folder instead of `internals`.

Reply:
classes have been purged.. it does make testing much easier.

Reply:
Dope, reading the PR now